@kiva/kv-components/vue/stories/KvToast.stories.js

```diff
@@ -0,0 +1,74 @@
+import KvButton from '../KvButton.vue';
+import KvToast from '../KvToast.vue';
+
+export default {
+	title: 'KvToast',
+	component: KvToast,
+	args: {
+		message: 'Sucessfully added to basket!',
+		persist: false,
+		type: 'confirmation',
+	},
+	argTypes: {
+		message: {
+			control: {
+				type: 'text',
+			},
+		},
+		type: {
+			options: ['confirmation', 'warning', 'error'],
+			control: {
+				type: 'select',
+			},
+		},
+		persist: {
+			control: {
+				type: 'boolean',
+			},
+		},
+	},
+};
+
+const Template = (args, {
+	argTypes,
+}) => ({
+	props: Object.keys(argTypes),
+	components: {
+		KvToast,
+		KvButton,
+	},
+	template: `
+		<div>
+			<kv-button @click="showToast(message, type, persist)">Show Toast</kv-button>
+
+			<!-- div below is a kludge for storybook docs -->
+			<div class="tw-fixed tw-inset-0 tw-pointer-events-none">
+				<div class="tw-fixed tw-z-toast tw-left-0 tw-right-0 tw-top-2 tw-pointer-events-auto">
```

Comment:
Should we have these style classes (ie tw-fixed, tw-z-toast and positiontion classes) embedded in the toast component? I think with those added, this component becomes something that can be used as a stand along component in some cases so graphql isn't a dependency.

Reply:
Since the toast may change positions depending on breakpoint and the height of the navigation, planning on handling positioning at the app level.